backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -42,14 +44,11 @@
   private PatientLinkService _pls;
   private final CurrentPatientContextHolder _patientContext;
 
-  public TestOrderService(
-      OrganizationService os,
-      DeviceTypeService dts,
-      TestOrderRepository repo,
-      PatientAnswersRepository parepo,
-      TestEventRepository terepo,
-      PersonService ps,
-      PatientLinkService pls,
+  public static final int DEFAULT_PAGINATION_PAGEOFFSET = 0;
+  public static final int DEFAULT_PAGINATION_PAGESIZE = 5000;
+
+  public TestOrderService(OrganizationService os, DeviceTypeService dts, TestOrderRepository repo,
```

Comment:
My IDE is a bit aggressive in applying formatting on save, which I generally like, but unfortunately causes unnecessary diffs. If this proves too hard to review, I'm happy to try to back out any purely formatting changes.